README.md

```diff
@@ -125,6 +127,9 @@ The service provider metadata used to ease configuration of the SAML SP in the I
   *Note*: All attributes can also be found in an array under `auth_hash[:extra][:raw_info]`,
   so this setting should only be used to map attributes that are part of the OmniAuth info hash schema.
 
+* `:soft` - Used to enable/disable the soft mode. When is set false (the
+  default), the saml validations errors will raise an exception. Setting this to true is useful when testing or in environment where you don't want to set up a real SAML server with signed requests.
```

Comment:
Can you change "the saml validations errors will" to "SAML validation errors will"?